web/ui/templates/targets.html

```diff
@@ -1,57 +1,72 @@
-{{define "head"}}<!-- nix -->{{end}}
+{{define "head"}}
+<link type="text/css" rel="stylesheet" href="{{ pathPrefix }}/static/css/targets.css?v={{ buildVersion }}">
```

Comment:
just to be consistent with alerts.html and alerts.css. Should it be in the one file?